site/en/docs/web-platform/view-transitions/index.md

```diff
@@ -375,6 +383,75 @@ The thumbnail now transitions into the main image. Even though they're conceptua
 
 The real code for this is a little more complicated than the simple example above, as it also handles the transition back to the thumbnail page. [See the source](https://glitch.com/edit/#!/simple-set-demos?path=6-expanding-image%2Fscript.js%3A15%3A17) for the full implementation.
 
+## Custom entry and exit transitions
+
+Look at this example:
+
+<style>
+  .enter-exit-video {
+    aspect-ratio: 1400/776;
+  }
+</style>
+
+<figure>
+  {% Video
+    playsinline="true",
+    src="video/CZmpGM8Eo1dFe0KNhEO9SGO8Ok23/uXSSraCaMIvu7nbWv0XS.mp4",
+    class="enter-exit-video",
+    loop="true",
+    muted="true",
+    controls="true"
+  %}
+  <figcaption>Entering and exiting sidebar. <a href="https://simple-set-demos.glitch.me/enter-exit-sidebar/">Minimal demo</a>. <a href="https://glitch.com/edit/#!/simple-set-demos?path=enter-exit-sidebar%2Fstyles.css%3A1%3A0">Source</a>.</figcaption>
+</figure>
+
+The side-bar is part of the transition:
+
+```css
+.side-bar {
+  view-transition-name: side-bar;
+  contain: layout;
+}
+```
+
+But, unlike the header in the previous example, the side-bar doesn't appear on all pages. If both states have the side-bar, the transition pseudo-elements look like this:
```

Comment:
I started changing side-bar to sidebar, but I realise you have that in your code and also further down refer to image-pair.

I think if you are referring to the name as code then use code font.

Reply:
I'll make it 'sidebar' in the code too.